src/test.rs

```diff
@@ -153,10 +153,10 @@ impl TestContext {
         )
     }
 
-    /// Lower `hugr` to LLVM, then JIT and execute the function named `entry` in
-    /// the inner module.
+    /// Lower `hugr` to LLVM, then JIT and execute the function named
+    /// `entry_point` in the inner module.
     ///
-    /// That function must take no arguments and return an `i64`.
+    /// That function must take no arguments and return an LLVM `i64`.
```

Comment:
Seems odd that the function must return an `i64` when this method is called `exec_hugr_u64`...

Reply:
`exec_hugr_u64` refers to the _hugr_ u64 that's returned, which is converted to an LLVM i64 (LLVM doesn't distinguish u/i in its type names). I was trying to help the confusion with this change, but we're not there yet... 🙃

Reply:
```suggestion
    /// That function must take no arguments and return a Hugr `u64` (compiled to an LLVM `i64`).
```

Reply:
No, hang on - Hugr doesn't distinguish these either, does it? So just `exec_hugr_i64`?